src/main/java/de/zalando/aruha/nakadi/Application.java

```diff
@@ -5,7 +5,13 @@
 
 @SpringBootApplication
 public class Application {
+
     public static void main(final String[] args) {
-        SpringApplication.run(Application.class, args);
+        FlowIdUtils.push(FlowIdUtils.generateFlowId());
```

Comment:
Why do we need to have an "application level" flow id? I thought it was something about the request, only. Does this comment make sense?